lib/block.rb

```diff
@@ -0,0 +1,35 @@
+require 'time'
+require_relative 'reservation'
+require_relative 'front_desk'
+require_relative 'room'
+
+class Block
+  attr_reader :block_start, :block_end, :rate, :block_rooms, :block_ID
```

Comment:
Minor note, you have a reader and writer for `block_ID`, instead maybe just make an accessor.